internal/models/cluster/extension/health.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright © 2022 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package extension
+
+import (
+	"encoding/json"
+)
+
+// VmwareTanzuManageV1alpha1ClusterExtensionHealth Health status of the deployed extension.
+//
+//   - HEALTH_UNSPECIFIED: Unknown.
+//   - HEALTHY: Healthy.
+//   - UNHEALTHY: Unhealthy.
+//
+// swagger:model vmware.tanzu.manage.v1alpha1.cluster.extension.Health
+type VmwareTanzuManageV1alpha1ClusterExtensionHealth string
+
+func NewVmwareTanzuManageV1alpha1ClusterExtensionHealth(value VmwareTanzuManageV1alpha1ClusterExtensionHealth) *VmwareTanzuManageV1alpha1ClusterExtensionHealth {
+	return &value
+}
+
+// Pointer returns a pointer to a freshly-allocated VmwareTanzuManageV1alpha1ClusterExtensionHealth.
+func (m VmwareTanzuManageV1alpha1ClusterExtensionHealth) Pointer() *VmwareTanzuManageV1alpha1ClusterExtensionHealth {
+	return &m
+}
+
+const (
+
+	// VmwareTanzuManageV1alpha1ClusterExtensionHealthHEALTHUNSPECIFIED captures enum value "HEALTH_UNSPECIFIED"
```

Comment:
Please ensure the linter checks pass, mostly it's due to the comments not ending with a period. Also, remove the unused functions that will cause linter failures.

Reply:
Will do